src/transaction/types.ts

```diff
@@ -74,6 +76,7 @@ export type TransactionContextType = {
   setIsToastVisible: (isVisible: boolean) => void; // A function to set the visibility of the transaction toast.
   setLifeCycleStatus: (state: LifeCycleStatus) => void; // A function to set the lifecycle status of the component
   setTransactionId: (id: string) => void; // A function to set the transaction ID.
+  transactions: Call[] | ContractFunctionParameters[]; // An array of transactions for the component.
```

Comment:
`calls` and `contracts` are only available as top-level props now, everything below `TransactionProvider` has been replaced with a single `transactions` union type for simplicity

Reply:
I like that